resources/views/statement/show.blade.php

```diff
@@ -14,7 +14,7 @@
     <h1 class="ecl-page-header__title ecl-u-type-heading-1 ecl-u-mb-l">Statement Details: {{$statement->uuid}}</h1>
 
     @if($statement->user && $statement->user->platform)
-        <x-infoline title="Sent by" :content="$statement->user->platform->name"></x-infoline>
+        <x-infoline title="Platform name" :content="$statement->user->platform->name"></x-infoline>
```

Comment:
I think we have a platform_name attribute mutator. It's all the same in the end.